VM/interface/interface.php

```diff
@@ -116,16 +116,27 @@
 		writeLog("==========interface end==========");
 		return;
 	}
-    if(!strcmp($CODE_LANGUAGE_TYPE,"C"))
-        $CODE_LANGUAGE_TYPE = 'c';
+    if(!strcmp($CODE_LAN_TYPE,"C"))
+        $CODE_LAN_TYPE = 'c';
     else
-        $CODE_LANGUAGE_TYPE = 'cpp';
-	$userCode = $userCode.$CODE_LANGUAGE_TYPE;
+        $CODE_LAN_TYPE = 'cpp';
+	$userCode = $userCode.$CODE_LAN_TYPE;
```

Comment:
prefer use
```
if (xxx) {
   @#$@#
} else {
   @#$@#
}
```
It will be more clear when reading code.

Also, please indent by same space/tab